UpdateProfile/handler.ts

```diff
@@ -101,7 +101,8 @@ export function UpdateProfileHandler(
       profilePayload,
       fiscalCode,
       emailChanged ? false : existingProfile.isEmailValidated,
-      existingProfile.servicePreferencesSettings.version
+      //TODO change with Michael's code
```

Comment:
Michael's code? 🙃

Reply:
while merging fix it with code from #171

Reply:
Nevermind, removed